File: h2h-PHP-SYMFONY/H2h/domain/MessageBreaker.php
<?php
namespace H2h\domain;
use \JsonSerializable;
class MessageBreaker implements JsonSerializable{


	private $pathClassMethodName;
	private $token;
    private $dateIncoming;

    public function __get($property) {
	    if (property_exists($this, $property)) {
	      return $this->$property;
	    }
  	}

  public function __set($property, $value) {
	  if (property_exists($this, $property)) {
	      $this->$property = $value;
	    }
    return $this;
  	}
  	public function jsonSerialize(){
        return get_object_vars($this);
    }
}

File: h2hell-core/src/main/java/com/highway2urhell/domain/MessageBreaker.java
package com.highway2urhell.domain;

public class MessageBreaker {
    private String pathClassMethodName;
    private String token;
    private String dateIncoming;
    private String parameters;

    public String getParameters() {
        return parameters;
    }

    public void setParameters(String parameters) {
        this.parameters = parameters;
    }

    public String getPathClassMethodName() {
        return pathClassMethodName;
    }

    public void setPathClassMethodName(String pathClassMethodName) {
        this.pathClassMethodName = pathClassMethodName;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

    public String getDateIncoming() {
        return dateIncoming;
    }

    public void setDateIncoming(String dateIncoming) {
        this.dateIncoming = dateIncoming;
    }

}
